### ISSUE TITLE
Edit news Item menu option missing on some themes and screen-widths

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#4445]
First noticed on Material

The narrow width side menu does not include the same option for "edit news item" as the main navigation:

<img width="297" alt="image" src="https://github.com/user-attachments/assets/c525f83f-b11a-465d-afcb-f3d5ba646ad4">

<img width="256" alt="image" src="https://github.com/user-attachments/assets/06be3b2d-a95f-493f-a950-6db1b0ca94c5">

To reproduce - navigate to a news item's page, and look at account menu for 'edit news item' option.  This appears on both Material and OLH in the main navigation, but does not appear on Clean or in any of the narrow side navs.

### Reviewing all themes:

| Theme | Narrow  | Main Navigation|
|---|---|---|
|Clean | :x: |:x: |
|Material | :x: | :white_check_mark:|
|OLH | :x:|:white_check_mark: |

**Comment by StephDriver** [comment]
Before working on this, I had another check through of where it showed up,  at the OLH issue has since been fixed.  Now for the rest!
| Theme | Narrow  | Main Navigation|
|---|---|---|
|Clean | :x: |:x: |
|Material | :x: | :white_check_mark:|
|OLH | ✅ |:white_check_mark: |
